cloud/gcp/modules/compute/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "google_cloud_run_service" "civiform_application_run_service" {
+  name     = "cloudrun-civiform"
+  location = var.region
+
+  ##TODO(@ktoor): Change to correct docker image. Will need to push the image to gcr through some mechanism.
+  template {
+    spec {
+      containers {
+        image = "us-docker.pkg.dev/cloudrun/container/hello"
+        resources {
+            requests = {
+                "cpu"    = "1"
+                "memory" = "512Mi"
+            }
+            limits = {
+                cpu      = "4"
+                "memory" = "7182Mi"
+            }
+        }
+        ports {
+          name = "http1"
+          container_port = var.http_port
+        }
+        env {
```

Comment:
curious how we pass env varialbes to the application itself or is this how?

Reply:
They will be set as environment variables in the container that the application can then use. Similar to how it's being used in aws right now

Reply:
https://github.com/seattle-uat/civiform/blob/main/infra/containers.yaml#L123

Reply:
i.e. these `env {` entries are how?

Reply:
Not sure I understand the question @bion

Reply:
Asking if this `env {`  block is how how environment variables are made available to the server

Reply:
Yes, it is.